.gitignore

```diff
@@ -25,3 +25,5 @@ data
 
 # ignore generated pem files from running the nuts-node executable.
 /*.pem
+
+nuts-node
```

Comment:
why?

Reply:
should you build it locally, it won't be checked in by accident. I'll add `/` so it only matches the file in the root